Library/AppDelegate.swift

```diff
@@ -34,11 +34,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	var window: UIWindow?
 	var coreDataStack = CoreDataStackImplementation.sharedInstance
 
-
-	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-		// Override point for customization after application launch.
-		return true
-	}
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
```

Comment:
Please use consistent indentation with the rest of the project (tabs instead of spaces).